publish-iis command

moozzyk opened the pull request.

dnfclas commented:
Hi __@moozzyk__, I'm your friendly neighborhood .NET Foundation Pull Request Bot (You can call me DNFBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by .NET Foundation and real humans are currently evaluating your PR.</p>

TTYL, DNFBOT;

Tratcher commented on src/Microsoft.AspNet.Tools.PublishIIS/Program.cs:
```diff
@@ -0,0 +1,73 @@
+                }
+
+                XDocument webConfigXml = null;
+                var webConfigPath = Path.Combine(publishFolder, "wwwroot", "web.config");
```
wwwroot can be renamed via the webroot parameter in hosting.json or command line.

moozzyk replied:
Yes, I am planning to handle this separately.

Tratcher replied:
Re: https://github.com/aspnet/Hosting/issues/449

Tratcher commented on src/Microsoft.AspNet.Tools.PublishIIS/project.json:
```diff
@@ -0,0 +1,22 @@
+  },
+
+  "dependencies": {
+    "NETStandard.Library": "1.0.0-rc2-23616",
```
fixed versions?

moozzyk replied:
Will fix this

Tratcher commented:
Please add a test case to make sure this bug in the prior implementation has been fixed: https://github.com/aspnet/dnx/issues/3062

moozzyk commented:
OK

moozzyk commented:
:up::date:
- unpinned package versions i project.json
- Moved from CommandLineUtils.Sources to CommandLineUtils
- Added test to verify that children of httpPlatform element are not removed (aspnet/dnx#3602)

Tratcher commented:
:shipit:
